framework/src/App.ts

```diff
@@ -199,7 +199,7 @@ export class App extends Extensible<AppConfig, AppMiddlewares> {
   }
 
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  async handleError(e: any, jovo?: Jovo): Promise<void> {
+  async handleError(e: Error, jovo?: Jovo): Promise<void> {
```

Comment:
I am not sure if we should change the type because it is theoretically possible that the value that is passed there is not an `Error`.

Reply:
Would `unknown` be the better choice here?

Reply:
I agree, `unknown` would be the better choice.